bug when trying to add a link when the app is not configured

I'm guessing. Google Play gives a few of these.
```
java.lang.RuntimeException: An error occured while executing doInBackground()
	at android.os.AsyncTask$3.done(AsyncTask.java:300)
	at java.util.concurrent.FutureTask.finishCompletion(FutureTask.java:355)
	at java.util.concurrent.FutureTask.setException(FutureTask.java:222)
	at java.util.concurrent.FutureTask.run(FutureTask.java:242)
	at android.os.AsyncTask$SerialExecutor$1.run(AsyncTask.java:231)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
	at java.lang.Thread.run(Thread.java:818)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method 'int java.lang.String.length()' on a null object reference
	at com.squareup.okhttp.FormEncodingBuilder.add(FormEncodingBuilder.java:38)
	at fr.gaulupeau.apps.Poche.network.WallabagService.getLoginRequest(WallabagService.java:175)
	at fr.gaulupeau.apps.Poche.network.WallabagService.executeRequestForResult(WallabagService.java:238)
	at fr.gaulupeau.apps.Poche.network.WallabagService.executeRequest(WallabagService.java:217)
	at fr.gaulupeau.apps.Poche.network.WallabagService.executeRequest(WallabagService.java:213)
	at fr.gaulupeau.apps.Poche.network.WallabagService.addLink(WallabagService.java:90)
	at fr.gaulupeau.apps.Poche.network.tasks.AddLinkTask.doInBackground(AddLinkTask.java:62)
	at fr.gaulupeau.apps.Poche.network.tasks.AddLinkTask.doInBackground(AddLinkTask.java:22)
	at android.os.AsyncTask$2.call(AsyncTask.java:288)
	at java.util.concurrent.FutureTask.run(FutureTask.java:237)
	... 4 more

```

--- a/app/src/main/java/fr/gaulupeau/apps/Poche/network/WallabagService.java
+++ b/app/src/main/java/fr/gaulupeau/apps/Poche/network/WallabagService.java
@@ -172,9 +172,10 @@ public class WallabagService {
     private Request getLoginRequest() {
         String url = endpoint + "/?login";
 
+        // TODO: maybe move null checks somewhere else
         RequestBody formBody = new FormEncodingBuilder()
-                .add("login", username)
-                .add("password", password)
+                .add("login", username != null ? username : "")
+                .add("password", password != null ? password : "")
 //                .add("longlastingsession", "on")
                 .build();
 

